documentation/meta/monitoring/cloudwatch_logs/index.md

```diff
@@ -0,0 +1,246 @@
+the API log group, you will not be able to narrow your query to exclude the
+nginx log streams explicitly. Instead, structure your query such that it will
+only match log events from the Django application (or vice-versa).
+
+### Logs Insights
+
+Logs Insights is very powerful. It is particular useful for deriving numerical
+and statistical data from both structured and unstructured logs, though the
+latter require additional parsing. Logs Insights has its own SQL-like query
+syntax,
+[the documentation for which](https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/CWL_QuerySyntax.html)
+should be the read before attempting to use Logs Insights for anything serious.
+
+The Logs Insights tutorials present sufficent examples for different types of
+queries that you may need to do. A good tip is to think of the Logs Insights
+query as a multi-step extract, transform, and load/aggregate tool. First,
+extract the specific log events that concern you. If you're querying to find the
+average ES query time logged in a particular format, first narrow the log
+streams down to just the service sending those logs. Next, use `parse` to
+extract the relevant information from the unstructured logs. Finally, use
+`stats` to aggregate the data and find the statistical information relevant to
+you. The following are two simple examples that can serve as starting points for
+querying structured and unstructured log data in Logs Insights for our
+application. Please note that logs insights queries are **not free**.
+[See the advice below on how to minimise costs](#logs-insights-queries-are-not-free).
+
+**Query for number of failed thumbnail requests per media type in the query
+period (parse unstructured logs)**:
+
+```
+# Filter down to only the django log streams (excluding nginx)
+# Filter to events that include "Failed Dependency"
+filter @logStream like 'ecs/django' and @message like 'Failed Dependency'
+
+# Extract the media type and UUID from the message
+| parse "/v1/*/*/thumb/" as mediaType, uuid
+
+# Aggregate the data to count
+| stats count_distinct(uuid) by mediaType as mediaTypeCount
+```
+
+**Query for the average timing of an API endpoint (leverage structured logs)**:
+
+```
+# Filter down to only the nginx log streams (excluding django)
+# Filter down to requests with "search" in the URL (stored in `request` field)
+filter @logStream like 'ecs/nginx' and request like "search"
+
+# Calculate the average request time for every 10 minute period within the query period
+| stats avg(request_time) by bin(10m) as averageSearchTime
+```
+
+In this example, `request` and `request_time` come directly the the structured
+JSON logs that our API Nginx container outputs. You can find the list of
+available fields in the
+[Nginx configuration](https://github.com//WordPress/openverse/blob/HEAD/api/nginx.conf.template#L10-L24).
+
+If you have access to Openverse's incident investigation reports, you will also
```

Comment:
Since all of this doc is only for the Openverse maintainers, we probably don't need this qualification ("if you have access").